src/paperless_tika/parsers.py

```diff
@@ -90,7 +90,7 @@ def convert_to_pdf(self, document_path, file_name):
         with open(document_path, "rb") as document_handle:
             files = {
                 "files": (
-                    file_name or os.path.basename(document_path),
+                    "convert." + file_name.split(".")[-1] or "convert" + os.path.splitext(document_path)[-1],
```

Comment:
I think `os.path.splitext` should handle file_name just fine as well, instead of splitting on a `.`.  The value will include an extension, if any

Reply:
Yes, that works